test/grid-rtl.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
+The complete set of authors may be found at http://polymer.github.io/AUTHORS
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
+-->
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>iron-list test</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+
+  <link rel="import" href="fixtures/helpers.html">
+  <link rel="import" href="fixtures/x-grid.html">
+</head>
+<body>
+
+
+  <test-fixture id="trivialList">
+    <template>
+      <x-grid dir="rtl" style="width: 300px;"></x-grid>
+    </template>
+  </test-fixture>
+
+<script>
+
+  suite('basic features', function() {
+    var list, container;
+
+    setup(function() {
+      container = fixture('trivialList');
+      list = container.list;
+    });
+
+
+    test('check horizontal rendering for rtl direction', function(done) {
+      container.data = buildDataSet(100);
+
+      flush(function() {
+        // Validate that items are being laid out from left to right.
+        for (var i = 0, rowIndex, indexInRow, itemsFromPreviousRows,
+             actualVisualIndex, expectedVisualIndex; i < 9; i++) {
+          rowIndex = getGridRowFromIndex(list, i);
```

Comment:
This test is failing because:
```js
function getGridRowFromIndex(grid, index) {
    return Math.ceil(index / grid._itemsPerRow);
  }

```

should be `Math.floor` instead of `Math.ceil`